lib/ipmi_backend_utils.pm

```diff
@@ -262,6 +262,18 @@ sub adjust_for_ipmi_xen {
     }
 }
 
+# Set boot from pxe via ipmitool
+sub set_pxe_boot {
+    while (1) {
+        my $stdout = ipmitool('chassis bootparam get 5');
+        last if $stdout =~ m/Boot Flag Valid[\d\D]*Force PXE/;
+        diag "setting boot device to pxe";
```

Comment:
I think you'd better use record-info here.

Reply:
Yes, record_info does help. I did not notice whether the machine boots from pxe when I once run the test with it.

Reply:
Will you add record_info ?

Reply:
yes, doing verificatioin run before push